rasa/core/policies/ensemble.py

```diff
@@ -46,6 +51,18 @@ def __init__(
 
         self._check_priorities()
 
+        if "MappingPolicy" not in policies:
```

Comment:
please put this into a separate function (e.g. `_check_for_important_policies` or something along these lines)